### PR TITLE
Manual Cherry-Pick of PR15542 to 202405: Fix t0 isolated d128u128s2 topo 202405

### PR DESCRIPTION
**Pull request by ccroy-arista**
### Description of PR
This PR contains the following changes, in order to achieve a functional t0-isolated-d128u128s2 topo:
- The field 'bp_interfaces' in the t0-isolated-d128u128s2 yml file is corrected to 'bp_interface' i.e. w/o the 's'.
- Added more VMs, to support the larger number of VMs needed for the topo.
- Fixed the synthesis of the MACs used in ansible roles so that it does not error out after 256 interfaces.

Additionally, this PR fixed the 'bp_interfaces' for t0-isolated-d128u128s1 yml file.

**This is a cherry-pick to 202405 branch for the following PR: https://github.com/sonic-net/sonic-mgmt/pull/15542**

Summary:
Fixes # (issue)

### Type of change
- [x] Bug fix
- [ ] Testbed and Framework(new/improvement)
- [ ] Test case(new/improvement)

### Back port request
- [ ] 202012
- [ ] 202205
- [ ] 202305
- [ ] 202311
- [x] 202405

#### What is the motivation for this PR?
To achieve a functional t0-isolated-d128u128s2 for use in sonic-mgmt tests.

#### How did you do it?
Made a series of fixes/updates as detailed in the summary above.

#### How did you verify/test it?
Confirmed that the testbed server can now be successfully configured as a result of these changes, and subsequently ran the sonic-mgmt test suite.

### CODE REVIEW

**Comment by ccroy-arista** [comment]
@r12f FYI

**Review by r12f — APPROVED** [review]

File: ansible/roles/test/files/helpers/change_mac.sh [hunk 1]
```diff
@@ -7,7 +7,7 @@ INTF_LIST=$(ls /sys/class/net | grep -E "^eth[0-9]+$")
 for INTF in ${INTF_LIST}; do
     ADDR="$(cat /sys/class/net/${INTF}/address)"
     PREFIX="$(cut -c1-15 <<< ${ADDR})"
-    SUFFIX="$(printf "%02x" ${INTF##eth})"
+    SUFFIX="$(printf "%02x" $((${INTF##eth}%256)))"
```
Comment (sdszhang) [inline ansible/roles/test/files/helpers/change_mac.sh:10]: This part needs to be updated to avoid duplication when more than 256.

Comment (ccroy-arista) [reply ansible/roles/test/files/helpers/change_mac.sh:10]: Thank you for catching that! I will push the update shortly.

Comment (ccroy-arista) [reply ansible/roles/test/files/helpers/change_mac.sh]: Update pushed.

**Review by sdszhang — APPROVED** [review]

**Pull request merged**